OpenSim/Examples/OptimizationExample_Arm26/OptimizationExample.cpp

```diff
@@ -160,7 +160,7 @@ int main()
         //Optimizer opt(sys, InteriorPoint);
 
         // Specify settings for the optimizer
-        opt.setConvergenceTolerance(0.1);
+        opt.setConvergenceTolerance(0.11);
```

Comment:
I am more concerned as to why the refactoring changed the answer than the additional time it spent.